homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/ball/Magic.java

```diff
@@ -0,0 +1,53 @@
+package com.tutrit.java.quickstart.homework.ball;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class Magic {
+    Logger log = LoggerFactory.getLogger("Magic Ball ");
+    Map<Integer, String> ball = new HashMap<>();
```

Comment:
privat

Reply:
Why?

Reply:
Best practice explicitly specify the access modifier for methods and varible of class

Reply:
fixed